docs/internal-documentation/maintenance/IssueLabels.md

```diff
@@ -3,52 +3,64 @@
 ## Label categories
 
 The table below defines the labels to be used when classifying issues and pull requests. There
-are three categories of labels defined:
+are four categories of labels defined:
 
-* Area that the issue covers
-* Priority of issue
-* Issue type
+* Area
+* Priority
+* Status
+* Type
```

Comment:
Yes, that should be included too. Updated with your suggestions except the colors.

1. What you suggested for pink does not look pink to me. I looked pink up in Wikipedia and took the code from there, `#FFC0CB` (https://en.wikipedia.org/wiki/Pink).
2. Changing the colors takes some time to do. Can you really motivate that change?

Reply:
How about calling it purple? The nice thing about this set of hues is that they're equally separated in the perceived color space (LCh). I believe it should be both easy to read and easy on the eye, and also usable for color deficient people. If we just pick a random color and add it to the existing palette, the resulting palette might not have these properties. If you like I can make the updates to the settings.

Reply:
I have looked at your proposal and I have created test labels [here] with both the current colors and your proposal. Labels `0-xxx` use the current colors or the new pink color from this PR. Labels `1-xxx` uses the colors that you propose.

If we were to create a nice looking page your colors would maybe be preferred, but here we want the colors to be distinct.
* Your `green` is not distinct. It is darker and it is harder to read the text. 
* Your `red` has a red tone, but I would not be sure to call it red. 
* Your `yellow` is not yellow to me. 
* Your `blue` is less distinct but could be fine. 
* Your `pink` is not pink, but it could be fine instead of pink if it is called purple (as you suggest).

We decided that the color for the new category `Versioning` should be pink, but if all agree we could change that to purple and use your color. Else I would prefer to keep the colors that we have in use. 

[here]: https://github.com/matsduf/zonemaster-issue-label-colors/labels?sort=name-asc